automobile/sumo-interface.md

```diff
@@ -57,6 +57,10 @@ field `controllerArgs` in order to customize the behavior of the interface:
 %end
 
 
+> **note** [Mac OSX]:
+On mac OSX, SUMO relies on X11. You need therefore to install [XQuartz](https://www.xquartz.org) (version 2.7.8 or later) for the interface to work.
```

Comment:
I not completely sure about the syntax we use in the documentation but I think should be ``Mac OS X``.